codegen/test/chart/conditional-sidecar/templates/deployment.yaml

```diff
@@ -197,7 +197,7 @@ spec:
   type: {{ $glooMgmtServer.serviceType }}
   ports:
   - name: grpc
-    port: {{ $glooMgmtServer.ports.grpc }}
+    port: {{ index $glooMgmtServer "ports" "grpc" }}
```

Comment:
so this is the only existing test chart that has a `Service` with parameterized ports?

Reply:
Yes, otherwise they should have been updated with autogen.